### PR TITLE
Enable tty output from Docker Actions

### PR DESCRIPTION
**Pull request by dicko2**
Thoughts?

Discussion in #808

### CODE REVIEW

File: src/Runner.Worker/Container/DockerCommandManager.cs [hunk 1]
```diff
@@ -197,7 +197,8 @@ public async Task<int> DockerRun(IExecutionContext context, ContainerInfo contai
 
             dockerOptions.Add($"--workdir {container.ContainerWorkDirectory}");
             dockerOptions.Add($"--rm");
-
+            dockerOptions.Add($"-t");
```
Comment (TingluoHuang) [inline src/Runner.Worker/Container/DockerCommandManager.cs:200]: you might want to add the `-t` to where the runner run `docker exec` as well.

Comment (dicko2) [reply src/Runner.Worker/Container/DockerCommandManager.cs:200]: in the public method here?

https://github.com/actions/runner/blob/6e2fa11b267e79fb553b1b759fbe041eb4db945f/src/Runner.Worker/Container/DockerCommandManager.cs#L301

Comment (TingluoHuang) [reply src/Runner.Worker/Container/DockerCommandManager.cs:200]: I added it for you. 😄

Comment (dicko2) [reply src/Runner.Worker/Container/DockerCommandManager.cs:200]: Should we use "-it" in both places?

**Comment by dicko2** [comment]
Good to merge @TingluoHuang ?

**Review by TingluoHuang — APPROVED** [review]

**Comment by everops-alex** [comment]
This was reverted in #977 (`v2.277.1`) -- the issue is, `teletype` was reverted, as well as the `-t` docker `tags` command.

Please add it back! 🙏 see details in #977 thread.

cc @chrispat

File: src/Runner.Worker/Container/DockerCommandManager.cs (hunk 1 above)
Comment (mary-doc) [reply src/Runner.Worker/Container/DockerCommandManager.cs:200]: Thanks all

**Pull request merged**